Doppler.Currency/Startup.cs

```diff
@@ -111,7 +111,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILoggerF
 
             app.UseSwaggerUI(c =>
             {
-                c.SwaggerEndpoint("/swagger/v1/swagger.json", "Doppler Currency API V1");
+                c.SwaggerEndpoint("v1/swagger.json", "Doppler Currency API V1");
```

Comment:
What do you think about removing the version completelly?

```suggestion
                c.SwaggerEndpoint("swagger.json", "Doppler Currency API V1");
```